pkg/aggregate/iterator.go

```diff
@@ -267,15 +268,27 @@ func (as *AggregateSet) updateCell(aggr AggrType, cell int, val float64) {
 // return the value per aggregate or complex function
 func (as *AggregateSet) GetCellValue(aggr AggrType, cell int) (float64, bool) {
 
-	if cell > as.maxCell || cell >= as.length || !as.validCells[cell] { // TODO: should >Len return NaN or Zero ?
+	if !isValidCell(cell, as) {
```

Comment:
you do not check `cell > as.maxCell ` is that by design?

Reply:
I see no reason for this check since maxCell is updated by adding a new cell